frontend/config/chains/1101.json

```diff
@@ -0,0 +1,15 @@
+{
+  "identifier": 1101,
```

Comment:
Don't we also need a config for the polygon goerli testnet?

Reply:
Good point. Will add it.